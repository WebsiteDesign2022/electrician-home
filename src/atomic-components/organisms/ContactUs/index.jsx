
import React from 'react';

import ContactDetailItem from '../../molecules/ContactDetailItem';

import './index.scss';

const ContactUs = () => (
    <div className="row contact-us">
        <div className="col-md-8 row form-group">
            <div className="row">
                <div className="col-sm-6">
                    <input 
                        id="name" 
                        className="form-control contact-input" 
                        type="text" 
                        placeholder="Name" 
                        aria-label="Name"/> 
                </div>
                <div className="col-sm-6">
                    <input 
                        id="email-address" 
                        className="form-control contact-input" 
                        type="text" 
                        placeholder="Email" 
                        aria-label="Email Address"/>
                </div>
            </div>

            <div className="row">
                <div className="col-md-12">
                    <textarea className="form-control contact-input" rows="5" placeholder="Message" id="message"></textarea>
                </div>
            </div>

            <div className="row justify-content-end">
                <div className="col-9"></div>
                <div className="col-3">
                    <button type="button" className="btn rounded-0 button-right submit">SUBMIT</button>
                </div>
            </div>
            
        </div>
        <div className="col-md-4">

            <ContactDetailItem
                title="Address"
                content="1234 Divi St., #1000, San Francisco, CA 29362"
            />

            <ContactDetailItem
                title="Phone"
                content="(346) 234 5675"
            />

            <ContactDetailItem
                title="Email"
                content="info@divelectric.com"
            />

        </div>
    </div>
);

export default ContactUs;