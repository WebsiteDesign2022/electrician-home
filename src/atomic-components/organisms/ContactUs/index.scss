.contact-us {
    background-image: linear-gradient(to right, grey, darkgrey);
    padding: 64px;
}

.form-control, .contact-input {
    border-radius: 0px;
    margin-bottom: 12px;
}

.btn.submit {
    color: white;
    background-color: orange;
    box-shadow: 10px 10px darkgrey;
}
