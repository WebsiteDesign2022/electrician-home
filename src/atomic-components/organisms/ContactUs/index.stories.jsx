import React from 'react';

import ContactUs from '.';

export default {
  title: 'Electrician-Home/organisms/ContactUs',
  component: ContactUs,
  parameters: {
    // More on Story layout: https://storybook.js.org/docs/react/configure/story-layout
    layout: 'fullscreen',
  },
};

const Template = (args) => <ContactUs {...args} />;

export const Primary = Template.bind({});
// More on args: https://storybook.js.org/docs/react/writing-stories/args
Primary.args = {

};
