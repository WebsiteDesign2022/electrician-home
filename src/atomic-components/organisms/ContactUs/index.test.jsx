import TestRenderer from 'react-test-renderer';

import ContactUs from '.';

describe("ContactUs", () => {
    test("renders ContactUs", () => {
        const contactUs = TestRenderer.create(
            <ContactUs/>
        );

        expect(contactUs.toJSON()).toMatchSnapshot();
    });
});
