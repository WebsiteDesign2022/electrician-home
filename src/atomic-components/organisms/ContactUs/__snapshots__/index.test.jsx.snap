// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactUs renders ContactUs 1`] = `
<div
  className="row contact-us"
>
  <div
    className="col-md-8 row form-group"
  >
    <div
      className="row"
    >
      <div
        className="col-sm-6"
      >
        <input
          aria-label="Name"
          className="form-control contact-input"
          id="name"
          placeholder="Name"
          type="text"
        />
      </div>
      <div
        className="col-sm-6"
      >
        <input
          aria-label="Email Address"
          className="form-control contact-input"
          id="email-address"
          placeholder="Email"
          type="text"
        />
      </div>
    </div>
    <div
      className="row"
    >
      <div
        className="col-md-12"
      >
        <textarea
          className="form-control contact-input"
          id="message"
          placeholder="Message"
          rows="5"
        />
      </div>
    </div>
    <div
      className="row justify-content-end"
    >
      <div
        className="col-9"
      />
      <div
        className="col-3"
      >
        <button
          className="btn rounded-0 button-right submit"
          type="button"
        >
          SUBMIT
        </button>
      </div>
    </div>
  </div>
  <div
    className="col-md-4"
  >
    <div
      className="feature"
    >
      <img
        src=""
      />
      <h5
        align="left"
      >
        Address
      </h5>
      <p
        align="left"
      >
        1234 Divi St., #1000, San Francisco, CA 29362
      </p>
    </div>
    <div
      className="feature"
    >
      <img
        src=""
      />
      <h5
        align="left"
      >
        Phone
      </h5>
      <p
        align="left"
      >
        (346) 234 5675
      </p>
    </div>
    <div
      className="feature"
    >
      <img
        src=""
      />
      <h5
        align="left"
      >
        Email
      </h5>
      <p
        align="left"
      >
        info@divelectric.com
      </p>
    </div>
  </div>
</div>
`;
